tools/dash/dash.bzl

```diff
@@ -47,7 +47,17 @@ def dash_license_checker(
         runtime_deps = [
             "@dash_license_tool//jar",
         ],
-        args = ["$(location :{}2dash)".format(name)],
+        # We'll build up "args" in the order: [ static options ] + select() + [ file last ]
+        args = [
+            # If we have any always-on flags, put them here
+        ] + select({
+            # If CI_BUILD=true, add "-review"
```

Comment:
I need to think about:
* Using "CI" from env, instead of a custom flag
* Using PERFORM_CLEARANCE instead of generic CI_BUILD
* Pass `-review` same as the other parameters, without select

Reply:
Third options sounds most reasonable? Especially since the 3 other parameters are passed directly anyway.

Reply:
I think I would have simply used -review. But that's only my gut feeling. So let's keep it as is. 
Works for now!
Doesn't matter whether we adjust it in the PR or later on demand - if there is ever such a demand.